README.md

```diff
@@ -73,6 +73,10 @@ development and community news:
 - Join the developer [mailing list].
 - Post on the [Mixxx forums][forums].
 
+## Distributed
```

Comment:
Distributed or Distribution?
I haven't checked how others name this section.

Reply:
Distributions also ?

Reply:
I propose to omit the extra section and instead add this badge just below the top-level header `# Mixxx`. This is the place where users will look first. Otherwise no one will take notice.

Reply:
How about the compact badge on the top level and optionally the detailed badge in a separate section at the end of the README?

Reply:
Checkout https://github.com/luzpaz/mixxx/blob/repology/README.md

Reply:
The small badge on top of the README works great for me :+1: Clicking on it gets you where you find all the details.

On the other hand the huge badge in `Distributed` is way too much. As I also mentioned before that this section's title doesn't fit with the namings of the other sections.

Reply:
pushed 1ddc39b
should be good to go